wireguard-install.sh

```diff
@@ -80,12 +100,21 @@ else
   ENDPOINT="$SERVER_PUB_IP:$SERVER_PORT"
 fi
 
+function wireguard-install() {
 # Install WireGuard tools and module
 if [[ "$OS" = 'ubuntu' ]]; then
     add-apt-repository ppa:wireguard/wireguard
     apt-get update
     apt-get install "linux-headers-$(uname -r)"
     apt-get install wireguard iptables
+elif [[ "$OS" = 'raspbian' ]]; then
```

Comment:
Please make a separate PR for that, thanks!

Reply:
master, we can't just make a separate PR for that only we would also have to move the distro code too.